### ISSUE TITLE
completion failed when typing “.unwrap()”

### ISSUE THREAD

**Issue by zzwxh** [intellij-rust/intellij-rust#9543]
## Environment

* **IntelliJ Rust plugin version:** 0.4.180.4932-222
* **Rust toolchain version:** stable-x86_64-pc-windows-msvc rustc 1.64.0 (a55dd71d5 2022-09-19)
* **IDE name and version:** CLion 2022.2.4 CL-222.4345.21
* **Operating system:** windows 11 

## Problem description 
completion failed when typing “.unwrap()”

## Steps to reproduce

```rust
use std::mem::size_of;
use windows::core::*;
use windows::Win32::Foundation::*;
use windows::Win32::Graphics::Gdi::{GetStockObject, GRAY_BRUSH, HBRUSH};
use windows::Win32::System::LibraryLoader::*;
use windows::Win32::UI::WindowsAndMessaging::*;

pub struct Window {
    window:HWND,
}

impl Window {
    pub fn create(title:String,width:i32,height:i32)->Window{
        unsafe {
            let style = WS_VISIBLE|WS_OVERLAPPEDWINDOW;
            let mut rect = RECT{
                left : 0,
                top : 0,
                right : width,
                bottom : height,
            };
            AdjustWindowRect(&mut rect,style,BOOL(0));
            let wc = WNDCLASSEXW{
                cbSize : size_of::<WNDCLASSEXW>() as u32,
                style : CS_HREDRAW|CS_VREDRAW,
                lpfnWndProc : Some(Window::window_proc),
                /*
                    hInstance : GetModuleHandleW(None).un... <- Completion Failed
                 */
                _hInstance : GetModuleHandleW(None).unwrap(),_   //<- Here
                lpszClassName : w!("ClassName").into(),
                hbrBackground : HBRUSH(GetStockObject(GRAY_BRUSH).0),
                ..Default::default()
            };
            println!("0:{}",GetLastError().0);
            RegisterClassExW(&wc);
            println!("1:{}",GetLastError().0);
            let hwnd = CreateWindowExW(
                WINDOW_EX_STYLE::default(),
                w!("ClassName"),
                &HSTRING::from(title).into(),
                style,
                CW_USEDEFAULT,CW_USEDEFAULT,
                rect.right-rect.left,
                rect.bottom-rect.top,
                None,None,
                GetModuleHandleW(None).unwrap(),
                None
            );
            println!("2:{}",GetLastError().0);
            assert_ne!(hwnd.0, 0);
            ShowWindow(hwnd,SW_SHOW);
            Window{
                window:hwnd,
            }
        }
    }
    pub fn pump_message(&self)->bool{
        unsafe {
            let mut msg = MSG::default();
            while PeekMessageW(&mut msg,None,0,0,PM_REMOVE).into() {
                if msg.message == WM_QUIT {
                    return false;
                }
                TranslateMessage(&msg);
                DispatchMessageW(&msg);
            }
        true
        }
    }
    extern "system" fn window_proc(window: HWND, message: u32, wparam: WPARAM, lparam: LPARAM)->LRESULT{
        unsafe{
            if message == WM_DESTROY {
                PostQuitMessage(0);
            }
            DefWindowProcW(window,message,wparam,lparam)
        }
    }
}

impl Drop for Window {
    fn drop(&mut self) {
        unsafe {
            UnregisterClassW(w!("ClassName"),GetModuleHandleW(None).unwrap());
            println!("3:{}",GetLastError().0);
        }
    }
}
```

**Comment by zzwxh** [comment]
my cargo.toml:

[package]
name = "LearnDx11rs"
version = "0.1.0"
edition = "2021"

# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html

[dependencies]

[dependencies.windows]
version = "0.42.0"
features = [
    "Win32_Foundation",
    "Win32_Graphics_Gdi",
    "Win32_System_LibraryLoader",
    "Win32_UI_WindowsAndMessaging",
]

**Comment by Undin** [comment]
For some reason, `::windows` cannot be resolved in definition of `GetModuleHandleW`, as a result, the plugin knows nothing about return type

**Comment by zzwxh** [comment]
but in vscode it works well

**Comment by dima74** [comment]
@zzwxh we have a big related to resolve of absolute paths, it will be fixed in #9545
